### PR TITLE
fewer words, easier reading

### PR DESCRIPTION
**Pull request by nicoaragon**
Incorporating feedback from [PR 9310](https://github.com/bazelbuild/bazel/pull/9310).

### CODE REVIEW

**Comment by laurentlb** [comment]
Oops, can you double-check? I don't see the expected changes

**Comment by nicoaragon** [comment]
My bad! changes are now in. Thank you!

File: site/docs/skylark/language.md [hunk 1]
```diff
@@ -105,10 +105,9 @@ illegal, and 2) `*args` and `**kwargs` arguments are not allowed.
 * Global variables are immutable.
 
 * `for` statements are not allowed at the top-level. Use them within functions
-  instead. In BUILD files, you may use list comprehensions.
+  instead. In BUILD files, you may use list comprehensions. That said, we prefer that you use macros, because we think they are easier to read.
```
Comment (laurentlb) [inline site/docs/skylark/language.md]: I'd remove this line (I know it was suggested by @brandjon).
We also want to limit the use of macros.

Comment (nicoaragon) [reply site/docs/skylark/language.md]: I removed it. Thank you!

**Pull request closed without merging**